### ISSUE TITLE
Jane can create a new Kabanero project under the Collection using Codewind.

### ISSUE THREAD

**Issue by ralanlittle** [kabanero-io/kabanero-collection#15]

**Comment by marikaj123** [comment]
Work will be done by August 12.

**Comment by Jhschube** [comment]
~This issue is currently blocked by https://github.com/kabanero-io/appsodyExtension/pull/28 (Which is in turn blocked by a separate PR)~

**Comment by marikaj123** [comment]
@Jhschube - What date do we need the fix to become available for us in order to help us complete the work on time?

**Comment by Jhschube** [comment]
closed as duplicate
